types/index.d.ts

```diff
@@ -0,0 +1,21 @@
+import { type FunctionComponentElement, type ProviderProps, type PropsWithChildren } from "react";
+
+interface Options {
+  debounce?: boolean,
+  events?: Record<string, Array<string>>
+  modelTypes?: Record<string, object>,
```

Comment:
This could probably be more specific than `string`, to limit it to `keyof Models`. Up to you if you think that's worth it though.